### PR TITLE
Replace old v1 params

### PR DESCRIPTION
**Pull request by debugwand**

### CODE REVIEW

File: server/models/article-v1.js [hunk 1]
```diff
@@ -16,7 +16,7 @@ module.exports = function(article) {
 	if (article.item.lifecycle.lastPublishDateTime) {
 		output.lastUpdated = article.item.lifecycle.lastPublishDateTime;
 	}
-	output.subheading = article.item.editorial.standFirst
+	output.standFirst = article.item.editorial.standFirst
```
Comment (laurieboyes) [inline server/models/article-v1.js]: Sorry, bit late, but is this camelCase intentional?

Comment (laurieboyes) [inline server/models/article-v1.js]: (also sorry this is such a mess - this file is one of the ones that will die with personalised feed)

Comment (laurieboyes) [inline server/models/article-v1.js]: Ah right actually we're probably not using that property anywhere 👍

Comment (debugwand) [inline server/models/article-v1.js]: no. I think this file is unused anyway but will update

**Pull request merged**